ui/shared/components/graph/IGV.jsx

```diff
@@ -42,7 +43,15 @@ class IGV extends React.PureComponent {
   }
 
   componentDidUpdate(prevProps) {
-    const { tracks } = this.props
+    const { tracks, locus } = this.props
+    if (this.browser && locus !== prevProps.locus) {
+      if (this.container) {
+        igv.removeBrowser(this.browser)
+        igv.createBrowser(this.container, { ...this.props }).then((browser) => {
```

Comment:
Maybe there is a better way to refresh the browser when the locus changed.

Reply:
probably, I suggest you read the IGV JS documentation to see